cms/envs/common.py

```diff
@@ -2079,9 +2087,17 @@
     'ROOT_PATH': 'sandbox',
 }
 
+############# CORS headers for cross-domain requests #################
+if FEATURES.get('ENABLE_CORS_HEADERS'):
+    CORS_ALLOW_CREDENTIALS = True
+    CORS_ALLOW_HEADERS = corsheaders_default_headers + (
+        'use-jwt-cookie',
+    )
+
 CORS_ORIGIN_WHITELIST = []
 CORS_ORIGIN_ALLOW_ALL = False
```

Comment:
This defaults to `False`. Do we need to set this value explicitly here as well?

Reply:
I think we list this out in the common.py to let others know about this specific setting.